controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -0,0 +1,267 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `/* eslint-disable guard-for-in */` and `/* eslint-disable no-restricted-syntax */` is generally discouraged as it can lead to potential bugs in the code. If you're using a `for...in` loop, consider refactoring your code to use `Object.keys()`, `Object.values()`, or `Object.entries()` instead.



<!-- This is an auto-generated comment by OSS CodeRabbit -->